src/duckling/canvas/drawing/util.ts

```diff
@@ -87,3 +103,22 @@ export function drawCanvasBorder(centerPosition : Vector, stageDimensions : Vect
         stageDimensions,
         graphics);
 }
+
+/**
+ * Converts an rgb color to a hex string
+ * @param  r red portion of the color (0-255)
+ * @param  g green portion of the color (0-255)
+ * @param  b blue portion of the color (0-255)
+ * @return hex string of the given rgb color (no # at the beginning)
+ */
+export function rgbToHex(r : number, g : number, b : number) : string {
```

Comment:
Needs Unit Test
Consider taking a Color object instead of the raw values.
Move to the color.ts file.